release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -291,6 +291,35 @@ Currently, it is not supported to use Active Directory Federation Services with
 
 For the full list of OIDC providers, see xref:../authentication/identity_providers/configuring-oidc-identity-provider.adoc#identity-provider-oidc-supported_configuring-oidc-identity-provider[Supported OIDC providers].
 
+[id="ocp-4-11-auth-pod-security-admission"]
+==== Pod security admission
+
+link:https://kubernetes.io/docs/concepts/security/pod-security-admission/[Pod security admission] is now enabled on {product-title}. 
+
+Clusters support pod security admission, as well as SCC admission, for pod admission. Pod security admission runs globally with "privileged" enforcement and "restricted" audit logging and API warnings.
+
+Clusters now monitor administrator-labeled namespaces for changes and map link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[pod security standards] to xref:../authentication/managing-security-context-constraints.adoc[security context constraints].
```

Comment:
`A controller monitors Service Account SCC-related permissions in user-created namespaces and automatically labels these namespaces with PSa `warn` and `audit` labels` (while keeping all the links :slightly_smiling_face:  )

Reply:
```suggestion
A controller monitors service account xref:../authentication/managing-security-context-constraints.adoc[SCC-related permissions] in user-created namespaces and automatically labels these namespaces with link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[pod security admission] `warn` and `audit` labels.
```